tox.ini

```diff
@@ -1,26 +1,29 @@
 [tox]
-envlist = py26, py27, py33, py34, py35, flake8
-
-[travis]
-python =
-    3.5: py35
-    3.4: py34
-    3.3: py33
-    2.7: py27
-    2.6: py26
-
-[testenv:flake8]
-basepython=python
-deps=flake8
-commands=flake8 pystac
+envlist = py34, py35, py36
 
 [testenv]
-setenv =
-    PYTHONPATH = {toxinidir}
+basepython =
```

Comment:
This setting is not needed. See https://tox.readthedocs.io/en/latest/config.html#factors-and-factor-conditional-settings